engines/native/sandbox.go

```diff
@@ -76,6 +86,39 @@ func newSandbox(b *sandboxBuilder) (*sandbox, error) {
 	return s, nil
 }
 
+func fetchContext(context, destdir string) error {
+	filename, err := util.Download(context, destdir)
```

Comment:
Add TODO to use the fetcher subsystem that I have PR for... This is exactly what it is intended for.

Add another TODO to use the cache subsystem when such system has been architected.